.github/workflows/node.js.yml

```diff
@@ -20,3 +20,49 @@ jobs:
     - run: npm install
     - run: npm run lint
     - run: npm run test
+
+  e2e:
+    runs-on: macos-11
+    env:
+      PLATFORM_VERSION: '15.2'
+      XCODE_VERSION: '13.2'
+      DEVICE_NAME: iPhone 11 Pro Max
+      NODE_VERSION: 14.x
+      PYTHON_VERSION: '3.7'
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: ${{ env.XCODE_VERSION }}
+    - run: xcrun simctl list
+    - run: brew install python@3.7
+    - run: |
+        brew install protobuf
+        brew install grpc
+
+        mkdir /tmp/protoc
+        cd /tmp/protoc
+        curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protoc-3.17.3-osx-x86_64.zip
+        unzip protoc-3.17.3-osx-x86_64.zip
+        chmod +x /tmp/protoc/bin/protoc
+        sudo ln -sf /tmp/protoc/bin/protoc /usr/local/bin/protoc
+
+        brew tap facebook/fb
+        brew install idb-companion || true
+
+        cd ~
```

Comment:
or `pushd`/`popd` may also help to keep the directory

Reply:
changed